src/shared/components/Settings/style.scss

```diff
@@ -157,7 +157,11 @@ $checkbox-bg-selected: $tc-dark-blue;
     .Select-placeholder,
     .Select-value-label,
     .Select-menu-outer {
-      @include merriweather-sans-regular;
+      font: {
+        size: 15px;
+        family: "Roboto", Helvetica, Arial, sans-serif;
```

Comment:
You should use the existing mixins for fonts instead of hardcoding the values.

Reply:
There was to existing mixin so I have to add a new on at fcb0bcd376cbedd377fac5540860edcb5cc47203